core/app/models/spree/order.rb

```diff
@@ -400,7 +400,7 @@ def deliver_order_confirmation_email
 
     # Helper methods for checkout steps
     def paid?
-      payment_state == 'paid' || payment_state == 'credit_owed'
+      %w(paid credit_owed).include?(payment_state)
```

Comment:
@mbj freezing all PR's/dev we're done cherry picking from 3.0 would be excessive and I'd way rather have the codebase moving forward. 
Doing large global changes are way more likely to cause conflicts (as they're touching more things), and while I agree they'll be in improvement to the codebase, waiting a bit would reduce friction in cherry picks. Its not that we're removing all conflicts, just attempting to minimize them for a set period of time.

@lightcap I'd be happy picking a date. I'd propose Dec 1st as a month seems like a reasonable time for people to have identified they want to move onto solidus from the 3.x branches and poke through the things important to them to attempt to get them PR'd in. It would be nice to grab as many of the commits from your performance patchset during this time as well.